app/assets/stylesheets/letter.scss

```diff
@@ -0,0 +1,37 @@
+$oil: #333333;
+$white: #ffffff;
+
+.letter {
+  background: $white;
+  box-shadow: 0 0 10px rgba(0,0,0,0.3);
+  margin: 26px auto 0;
+  width: 90%;
```

Comment:
Properties should be ordered background, box-shadow, margin, min-height, padding, position, width